core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -754,15 +754,15 @@ class GroupCoordinator(val brokerId: Int,
     }
   }
 
-  def handleFetchOffsets(groupId: String, partitions: Option[Seq[TopicPartition]] = None):
+  def handleFetchOffsets(groupId: String, requireStable: Boolean = false, partitions: Option[Seq[TopicPartition]] = None):
```

Comment:
If we expect all the callers to set `requireStable` we should not set a default value as it is bug-vulnerable.